modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/JraftServerImpl.java

```diff
@@ -301,6 +302,12 @@ public Path getServerDataPath(String groupId) {
     /** {@inheritDoc} */
     @Override
     public synchronized boolean startRaftGroup(String groupId, RaftGroupListener lsnr, @Nullable List<Peer> initialConf) {
+        return startRaftGroup(groupId, null, lsnr, initialConf);
+    }
+
+    /** {@inheritDoc} */
+    public synchronized boolean startRaftGroup(String groupId, RaftGroupEventsListener evLsnr,
```

Comment:
Should we add this to the `org.apache.ignite.internal.raft.server.RaftServer` and add `@Override` and new javadoc with  `RaftGroupListener lsnr` described?